src/wp-includes/embed.php

```diff
@@ -902,6 +902,7 @@ function wp_filter_oembed_iframe_title_attribute( $result, $data, $url ) {
  * Only filters 'rich' and 'video' response types.
  *
  * @since 4.4.0
+ * @todo This needs to be refactored to leverage the HTML API.
```

Comment:
Is this going to be done as part of this PR? If not, this should be removed before committing.

Reply:
There's a similar todo also committed in core:

https://github.com/WordPress/wordpress-develop/blob/3adbaf8ec020d653dd7e6c949a8b51073e726365/src/wp-includes/media.php#L2218

Reply:
```suggestion
```